english/run/run.md

```diff
@@ -7,4 +7,5 @@ BEEP! BEEP! BEEP! You wake up, and it's Tuesday morning... 5am.
 You get up, to go to your dead-end job to pay for the mountain of student loans your acquired
 while seeking that degree in Lego Engineering.
 You realize Lego Engineering may not have been the best choice in majors...
-But first, let me take a selfie.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.